### PR TITLE
Publicly Verifiable Covert MPC & Circuit-PSI

### PR DESCRIPTION
**Pull request by freepengui**
**Type of change**

- [x] Add new papers (Please tell us why you think this paper is awesome!)
- [ ] Fix the category of an existing paper/papers (Please tell us the reasons)
- [ ] Add a new tool/primitive/application with a new markdown page (Thank you! Also, please tell us more about this awesome thing!)

**Description**

> Please include a summary of the change and the relevant motivation.
在MPC中提交的两篇都是关于从半诚实安全mpc到公开可验证covert安全的mpc编译器
在PSI中提交的一篇关于具有线性通信复杂度的电路PSI论文

### CODE REVIEW

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Benny Pinkas, Mike Rosulek, Ni Trieu, Avishay Yanai*
   Crypto 2019, [eprint](https://eprint.iacr.org/2019/634), PRTY19
 
+- Efficient Circuit-based PSI with Linear Communication
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:72]: 这里可以在尾部加两个空格吗？会让 markdown 渲染的更好看一些

File: docs/awesome-pets/papers/applications/set/psi.md [hunk 2]
```diff
@@ -69,6 +69,10 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
...
+  *Benny Pinkas, Thomas Schneider, Oleksandr Tkachenko, Avishay Yanai*
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/applications/set/psi.md:73]: Same for this line

File: docs/awesome-pets/papers/tools/mpc.md [hunk 3]
```diff
@@ -266,6 +266,14 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Marcel Keller*
   CCS 2020, [eprint](https://eprint.iacr.org/2020/521), Kel20
 
+- Generic Compiler for Publicly Verifiable Covert Multi-Party Computation
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/tools/mpc.md:269]: ditto

File: docs/awesome-pets/papers/tools/mpc.md [hunk 4]
```diff
@@ -266,6 +266,14 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
...
+  *Sebastian Faust, Carmit Hazay, David Kretzler, and Benjamin Schlosser*
+  EUROCRYPT 2021, [eprint](https://eprint.iacr.org/2021/251), FHKS21
+  
+- Black-Box Transformations from Passive to Covert Security with Public Verifiability
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/tools/mpc.md:273]: ditto

**Review by Jamie-Cui — CHANGES_REQUESTED** [review]
LGTM

File: docs/awesome-pets/papers/tools/mpc.md [hunk 5]
```diff
@@ -266,6 +266,14 @@ Note: one paper may be included in several categories (e.g. a paper may introduc
   *Marcel Keller*
   CCS 2020, [eprint](https://eprint.iacr.org/2020/521), Kel20
 
+- Generic Compiler for Publicly Verifiable Covert Multi-Party Computation  
+  *Sebastian Faust, Carmit Hazay, David Kretzler, and Benjamin Schlosser*  
+  EUROCRYPT 2021, [eprint](https://eprint.iacr.org/2021/251), FHKS21  
+    
+- Black-Box Transformations from Passive to Covert Security with Public Verifiability   
+  *Ivan Damg˚ard, Claudio Orlandi, and Mark Simkin*
```
Comment (Jamie-Cui) [inline docs/awesome-pets/papers/tools/mpc.md:274]: 还有这里也请加一下 thx

**Comment by freepengui** [comment]
is this ok?

**Review by Jamie-Cui — APPROVED** [review]
LGTM

**Pull request merged**